pages/index.tsx

```diff
@@ -64,12 +71,23 @@ export async function getServerSideProps() {
     `${process.env.REST_API}/v1/public/videos/12/600`
   ).then((res) => res.json());
 
+  let liveFallback = await fetch(
+    `${process.env.REST_API}/v1/public/playout/channel`
+  )
+    .then((res) => res.json())
+    .then((e) => {
+      let keyedData = {};
+      keyedData[`${process.env.REST_API}/v1/public/playout/channel`] = e;
+      return keyedData;
+    });
```

Comment:
I'm not too sure what you are trying to do here since it looks like there will only be one key. Are you wanting to map the response and use the channel URL name as a key perhaps or is this just a SWR thing? Also, can we cast to the channel type at any point here?

Reply:
from what I know, the key is an SWR thing, it wasn't working before I copied the docs and did that anyway (although I might've been making a dumb mistake), can try adding a cast in the SSR portion 8861d838338d6119c58fdd6f537dc987dd859010

Reply:
Interesting